src/common/obj_bencher.cc

```diff
@@ -217,7 +218,8 @@ void *ObjBencher::status_printer(void *_bencher) {
 
 int ObjBencher::aio_bench(
   int operation, int secondsToRun,
-  int concurrentios, size_t op_size, size_t object_size,
+  int concurrentios,
+  uint64_t op_size, uint64_t object_size,
```

Comment:
just want to note that size_t on arm is 32bit. if people want to reuse the bench after upgrade, he/she need to nuke the bench data.